adapters/amx/amx.go

```diff
@@ -0,0 +1,197 @@
+package amx
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+const vastImpressionFormat = "<Impression><![CDATA[%s]]></Impression>"
+const vastSearchPoint = "</Impression>"
+const nbrHeaderName = "x-nbr"
+const adapterVersion = "pbs1.0"
+
+// AMXAdapter is the AMX bid adapter
+type AMXAdapter struct {
+	endpoint string
+}
+
+// NewAMXBidder creates an AMXAdapter
+func NewAMXBidder(endpoint string) *AMXAdapter {
+	endpointURL, err := url.Parse(endpoint)
+	if err != nil {
+		glog.Fatalf("invalid endpoint provided to AMX: %s, error: %v", endpoint, err)
+		return nil
+	}
+
+	qs, _ := url.ParseQuery(endpointURL.RawQuery)
```

Comment:
Please handle the error from `url.ParseQuery` in a similar manner as you did with `url.Parse`.